uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -43,6 +45,47 @@
         }
     }
 
+   .logout-launcher,.notification-icon,.portal-greeting,.login-launcher{
+        display: block;
+        float: right;
+        margin-right: 5px;
+        margin-top: 12px;
```

Comment:
Yep, I remember, I've struggle a bit to make all these different elements (inline block elements and inline blocks elements nested in block elements) perfectly align horizontally in the bar. :)